sc-memory/sc-core/sc-store/sc-fs-storage/sc_fs_storage.h

```diff
@@ -64,17 +66,32 @@ sc_bool sc_fs_storage_get_sc_links(const sc_char * sc_string, sc_addr ** links,
  * @param sc_substr A key substring
  * @param[out] links A pointer to sc-links
  * @param[out] size A sc-links size
- * @returns SC_TRUE, if sc-links exist.
+ * @returns SC_TRUE, if such sc-links exist.
  */
 sc_bool sc_fs_storage_get_sc_links_by_substr(const sc_char * sc_substr, sc_addr ** links, sc_uint32 * size);
 
+/*! Gets sc-strings from file system storage by it substring content.
+ * @param sc_substr A key substring
+ * @param[out] links A pointer to sc-strings array
+ * @param[out] size A sc-strings array size
+ * @returns SC_TRUE, if such sc-strings exist.
+ */
+sc_bool sc_fs_storage_get_sc_strings_by_substr(const sc_char * sc_substr, sc_char *** strings, sc_uint32 * size);
+
 /*! Gets sc-link content string with its size.
  * @param addr A sc-link
  * @param[out] sc_string A content string
  * @param[out] size A content string size
  */
 void sc_fs_storage_get_sc_string_ext(sc_element * element, sc_addr addr, sc_char ** sc_string, sc_uint32 * size);
 
+/*! Removes sc-link content string from file system storage.
+ * @param element A sc-link element
+ * @param addr A sc-link addr
+ * @returns SC_TRUE, if such sc-string exists.
+ */
+sc_bool sc_fs_storage_remove_sc_string(sc_element * element, sc_addr addr);
```

Comment:
This method is not clear for me with this name.
Why we talk about sc-string not about the contents of the link? 
As I understood, method just clear link content and can be named as `sc_fs_storage_remove_link_content` or `sc_fs_storage_clean_link_content`